utils/utils.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+
+exports.checkExists = (article_id) => {
```

Comment:
Firstly, I'm seeing no tests for this util - don't forget your TDD on everything!

Secondly, checkExists is a bit vague. What are you checking the existence of?

Thirdly, what's the purpose of this util? Do you really need to use it in the way you've got it implemented at the moment? Is there an easier way to check for the existence of an article?